source/fundamentals/crud/read-operations/skip.txt

```diff
@@ -0,0 +1,60 @@
+=====================
+Skip Returned Results
+=====================
+
+.. default-domain:: mongodb
+
+Use ``skip()`` to specify how many documents to omit from the beginning
+of the list of returned documents for a read operation. You can combine
```

Comment:
Can you use skip on an aggregation? And do aggregations fit in the classification of the aforementioned read operations? Could you provide info about that here?